src/main/webapp/course.html

```diff
@@ -212,6 +212,14 @@ <h1 class="p-4 text-center">Add Your Review</h1>
         >
           Submit
         </button>
+        <div id="message-rating"></div>
+        <button
+          id="latest-rating"
+          type="button"
+          class="btn btn-lg btn-outline-success"
+        >
+          Autofill Form With Previous Rating.
```

Comment:
Nit: It's not technically *auto*filling if you have to push a button to do it right? : P

Reply:
Hahaha, you are right! Changing name now!